.eslintrc.js

```diff
@@ -42,7 +43,7 @@ module.exports = {
             '@typescript-eslint/switch-exhaustiveness-check': ['warn'],
             '@typescript-eslint/no-unused-expressions': ['warn'],
             '@typescript-eslint/no-unused-vars': ['warn'],
-            '@typescript-eslint/no-use-before-define': ['warn'],
+            "no-use-before-define": ["error", { "functions": false, "classes": true, "variables": true }],
```

Comment:
use ' instead of "
soit passer à 'warn' plutôt que 'error', soit corriger toutes les erreurs introduites alors :D

Reply:
Je propose plutôt de le BUTTER :D
J'ai pas trouvé un endroit bien ou j'me dit "ah merci oui!" (mais les choses bien passent tristement inaperçue parfois)

Reply:
allez agreed, j'te laisse résoudre !